src/QirRuntime/test/QIR-static/qir-test-arrays.qs

```diff
@@ -33,9 +35,9 @@ namespace Microsoft.Quantum.Testing.QIR
         if (dummy)
         {
             let res1 = TestControlled();
-            //Q# compiler crashes if both TestControlled and TestPartials are enabled
-            //let res2 = TestPartials(17, 42);
+            let res2 = TestPartials(17, 42);
```

Comment:
FYI: For me the compiler didn't crash.

Reply:
You are probably on a newer version of the compiler, which means whatever bug it was, it's fixed. Good :)